frontend/src/pages/discover/today.vue

```diff
@@ -105,11 +91,25 @@ export default {
         merged: true,
         month: this.filter.month,
         day: this.filter.day,
+        order: "newest",
```

Comment:
So the results I see with this MR are in line with this code. The `order` that is rendered is `newest`.